apps/ehr/src/features/in-person/routing/InPersonRouting.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import useEvolveUser from 'src/hooks/useEvolveUser';
+import { AppTypeProvider } from 'src/shared/contexts/useAppFlags';
+import { InPersonLoader } from '../components/InPersonLoader';
+import { InPersonNavigationProvider, useInPersonNavigationContext } from '../context/InPersonNavigationContext';
+import { InPersonLayout } from '../layout/InPersonLayout';
+
+const InPersonRouting: React.FC = () => {
+  const currentUser = useEvolveUser();
+  const { availableRoutes = [], isLoading = true } = useInPersonNavigationContext() || {};
+
+  if (!currentUser || isLoading) {
+    return <InPersonLoader />;
+  }
+
+  // todo: 'permissions.view' was hardcoded to 'true' previously, should we use useAppointmentAccessibility to check if the user has access to the appointment?
+  // if (!permissions.view) {
```

Comment:
I commented out this code and left a TODO - it was never executing before because `view` was hardcoded to `true`. If we can use `useGetAppointmentAccessibility` here, i think we should create a task for this improvement.